lib/Doctrine/Record/Filter/Standard.php

```diff
@@ -34,24 +34,22 @@
 class Doctrine_Record_Filter_Standard extends Doctrine_Record_Filter
 {
     /**
-     * filterSet
-     * defines an implementation for filtering the set() method of Doctrine_Record
+     * @param string $propertyOrRelation
      *
-     * @param mixed $name                       name of the property or related component
+     * @thrown Doctrine_Record_UnknownPropertyException
      */
-    public function filterSet(Doctrine_Record $record, $name, $value)
+    public function filterSet(Doctrine_Record $record, $propertyOrRelation, $value)
```

Comment:
Snake case preferred again
```suggestion
    public function filterSet(Doctrine_Record $record, $property_or_relation, $value)
```